QA_Robot_Practice/services/LoginService.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Documentation    This is implementing of the login functionality
+Resource         ../pages/LoginPage.robot
+Resource         ../pages/MainPage.robot
+Resource         ../globalConfig/GlobalSettings.robot
+
+*** Keywords ***
+Open login page
+    Open browser in login
+
+Sign out Clarabridge
```

Comment:
It should consist of two navigator page actions: Open Dropdown [Name] and Sign Out